ci/verify_current_targets/CMakeLists.txt

```diff
@@ -34,6 +33,11 @@ set(ga_libraries
     spanner
     storage)
 
+# Define experimental libraries. The only difference is the name of the CMake
+# targets.
+set(experimental_libraries # cmake-format: sortable
+                           bigquery_rest)
```

Comment:
Shouldn't we then test these too:

https://github.com/googleapis/google-cloud-cpp/blob/c97e402abc4379cc8e72dc4ceede6140e0835aff/cmake/GoogleCloudCppFeatures.cmake#L26-L31

Reply:
> Shouldn't we then test these too

We are only sampling a few of the libraries to verify.

With that being said, adding `pubsublite` wouldn't hurt. And we should just GA `apikeys`. (I'll do it during my operator week)